src/administration/Administration.Service/BusinessLogic/InvitationBusinessLogic.cs

```
@@ -108,4 +109,10 @@ private async Task TriggerProcessStepInternal(Guid processId, ProcessStepTypeId
         context.FinalizeProcessStep();
         await _portalRepositories.SaveAsync().ConfigureAwait(ConfigureAwaitOptions.None);
     }
+
+    public async Task<CompanyInvitationDetails> GetApplicationAndCompanyDetails(Guid invitationId)
+    {
+        var companyInvitation = await _portalRepositories.GetInstance<ICompanyInvitationRepository>().GetInvitationCompanyAndApplicationId(invitationId);
```

In my opinion there is some error handling missing.

1. CompanyInvitation does not exist for the given id - should result in a NotFoundException (404)
2. this one might be discussable - If applicationId and companyId is null why return it? In my opinion we should return a 400 Exception here. At least the applicationId should always be set.

@MaximilianHauer what's your opinion the point 2?